tests/testthat/test-2mergeSEs.R

```diff
@@ -184,10 +181,9 @@ test_that("mergeSEs", {
     expect_equal( col_data1[rownames, colnames], col_data )
     
     # CHECK LEFT JOIN ##############################################
-    tse <- mergeSEs(list(tse1[11:20, 1:13], tse1[10:50, 7:20]), 
+    tse <- mergeSEs(list(tse1[11:20, 1:10], tse1[11:20, 1:10]), 
                        join = "left", collapse.cols = TRUE)
-    expect_true( all(dim(tse) == c(10, 20)) )
-    expect_equal( rowTree(tse), rowTree(tse1) )
```

Comment:
Instead of modifying the tests to match the code, we should modify the code to match the tests

For example, here is no clear reason why the tests did not work anymore. The changes in code should had only affect the tree, not the dimensions.